### ISSUE TITLE
need better citation for CodeMeta project

### ISSUE THREAD

**Issue by danielskatz** [codemeta/codemeta#186]
The suggested citation in the readme is
> You can cite the CodeMeta schema and project as:

> *Matthew B. Jones, Carl Boettiger, Abby Cabunoc Mayes, Arfon Smith, Peter Slaughter, Kyle Niemeyer, Yolanda Gil, Martin Fenner, Krzysztof Nowak, Mark Hahnel, Luke Coy, Alice Allen, Mercè Crosas, Ashley Sands, Neil Chue Hong, Patricia Cruse, Daniel S. Katz, Carole Goble.* 2017. __CodeMeta: an exchange schema for software metadata. Version 2.0.__ KNB Data Repository. [doi:10.5063/schema/codemeta-2.0](https://doi.org/10.5063/schema/codemeta-2.0)

But this is not a very good link for someone who want to cite the project, since the DOI doesn't point to a landing page with useful information, but rather points directly to a datafile.

**Comment by cboettig** [comment]
Agreed.  we should write a paper about the project as well.  Partly I've been hesitant to prioritize that yet since it's harder to update papers than to update software, and we've had a good bit of evolution on the standard.  But that seems to have settled down now and a paper is clearly overdue...  Hopefully this semester...

**Comment by danielskatz** [comment]
Wasn't there a fairly good start to a paper that we worked on in Portland at FORCE2016?

**Comment by cboettig** [comment]
There was a nice start outlining the objectives of codemeta, but there was no JSON-LD schema at the time, and I think the paper ought to give some sense of how to actually create and use `codemeta.json`.

**Comment by moranegg** [comment]
I would like to cite the CodeMeta project and in particular the crosswalk table, but I see that the DOI on the repository is directly pointing to the schema. What should I do?

**Comment by mbjones** [comment]
@moranegg For now, I think the citation below is your best option, as we don't have another at the moment:

> You can cite the CodeMeta schema and project as:

> Matthew B. Jones, Carl Boettiger, Abby Cabunoc Mayes, Arfon Smith, Peter Slaughter, Kyle Niemeyer, Yolanda Gil, Martin Fenner, Krzysztof Nowak, Mark Hahnel, Luke Coy, Alice Allen, Mercè Crosas, Ashley Sands, Neil Chue Hong, Patricia Cruse, Daniel S. Katz, Carole Goble. 2017. CodeMeta: an exchange schema for software metadata. Version 2.0. KNB Data Repository. doi:10.5063/schema/codemeta-2.0

**Comment by moranegg** [comment]
Thanks @mbjones !
I will add the following identifier to the citation if you don't mind:
[swh:1:dir:39c509fd2002f9e531fb4b3a321ceb5e6994e54a;origin=https://github.com/codemeta/codemeta](https://archive.softwareheritage.org/swh:1:dir:39c509fd2002f9e531fb4b3a321ceb5e6994e54a;origin=https://github.com/codemeta/codemeta/)
